tests/test_component/Sources/test_component/test_component+ABI.swift

```diff
@@ -1086,6 +1086,17 @@ public enum __ABI_test_component {
             }
         }
 
+        internal func VectorAsIterableImpl(_ value: test_component.AnyIVector<String>?) throws -> test_component.AnyIIterable<String>? {
+            let (result) = try ComPtrs.initialize { resultAbi in
```

Comment:
Why does result have parens?

Reply:
OK.  I would have thought that would make it a tuple of one?  I dunno.  If it works leave it.

Reply:
I think a tuple of one and a scalar are the same thing.

Reply:
unless they are not and the `let (x)` syntax deconstructs a tuple of one into a scalar declaration